csharp/src/Glacier2/DocGen.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) ZeroC, Inc.
+
+// Mark some generated server-side classes as EditorBrowsableState.Never so that docfx does not generate documentation
+// pages for them.
+
+using System.ComponentModel;
+
+namespace Glacier2;
+
+#pragma warning disable SA1601 // PartialElementsMustBeDocumented
+
+[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Isn't this attribute to avoid the classes to show up in code completion? If we just want to filter them in docfx I think it can be done in the docfx configuration.

Reply:
We really want both.